recipe/meta.yaml

```diff
@@ -15,14 +15,16 @@ source:
     - CMakeLists.patch  # [win and vc<14]
 
 build:
-  number: 1
+  number: 1001
 
 requirements:
   build:
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
     - cmake
     - pkg-config           # [unix]
+    - xorg-libx11          # [linux]
+    - xorg-libxext          # [linux]
```

Comment:
These should be in host.

Not sure about needing libxext, but maybe.

Reply:
One step further, but the build process still fails on missing libraries, alas:
```
/home/conda/feedstock_root/build_artifacts/glew_1546210395852/_build_env/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.3.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: cannot find -lGL
/home/conda/feedstock_root/build_artifacts/glew_1546210395852/_build_env/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.3.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: cannot find -lGLU
```

Reply:
Think that requires the Mesa CDTs. Not sure if we will want to move away from those in the future, but they are being used currently for other packags as well. Plus we need to sort out some things with `mesalib` before we can switch to that. Have pushed some commits to use the Mesa CDTs here.

xref: https://github.com/conda-forge/mesalib-feedstock/issues/16
xref: https://github.com/conda-forge/mesalib-feedstock/pull/17